config/routes.rb

```diff
@@ -3,5 +3,11 @@
     get 'drivers(.:format)',
         :to => 'api/v1/trucking/drivers#index',
         :defaults => { :format => 'json' }
+    get 'driver_commissions_history(.:format)',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.